From 980f1a8355e1e7e6a02a0c90e39b84c35215d041 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 14 Feb 2023 15:53:20 -0800
Subject: [PATCH] Customize for Clear Linux

For Clear Linux, control where built modules are installed and don't
move originals out of the tree, so swupd repair doesn't break DKMS.
---
 dkms.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dkms.in b/dkms.in
index 1cd55b157a24..3fdc3c808f62 100644
--- a/dkms.in
+++ b/dkms.in
@@ -424,6 +424,9 @@ override_dest_module_location()
     arch*)
         echo "/updates/dkms" && return
         ;;
+    clear-linux-os*)
+        echo "/updates/dkms" && return
+        ;;
     *)
         ;;
     esac
-- 
2.46.2

